easy_phi/app.py

```diff
@@ -33,16 +34,6 @@
 
 define("default_format", default='json')
 
-settings = {
```

Comment:
moved to line 261 where configuration from config file and command line already parsed